packages/pigeon/lib/java_generator.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'ast.dart';
+import 'generator_tools.dart';
+
+/// Options that control how Java code will be generated.
+class JavaOptions {
+  /// The name of the class that will house all the generated classes.
+  String className;
+
+  /// The package where the generated class will live.
+  String package;
+}
+
+void _writeHostApi(Indent indent, Api api) {
+  assert(api.location == ApiLocation.host);
+  indent.writeln('/** Generated interface from pigeon. */');
```

Comment:
Be consistent on capitalization. 
It's min case here and uppercase below.

Reply:
done.